extensions/oauth/src/main/java/org/mule/extension/oauth2/internal/DeferredExpressionResolver.java

```diff
@@ -45,4 +46,15 @@ public DeferredExpressionResolver(MuleContext muleContext) {
     }
   }
 
+  public <T> String getExpression(ParameterResolver<T> expr) {
```

Comment:
That is already working. What would be better for this extension would be to support something like the `literal`, but that was already ruled out.